src/stream/src/executor/global_simple_agg.rs

```diff
@@ -92,27 +91,14 @@ impl<S: StateStore> SimpleAggExecutor<S> {
     pub fn new(
         input: Box<dyn Executor>,
         agg_calls: Vec<AggCall>,
-        keyspace: Vec<Keyspace<S>>,
         pk_indices: PkIndices,
         executor_id: u64,
         key_indices: Vec<usize>,
+        state_tables: Vec<StateTable<S>>,
     ) -> Result<Self> {
         let input_info = input.info();
         let schema = generate_agg_schema(input.as_ref(), &agg_calls, None);
 
-        // Create state tables for each agg call.
-        let mut state_tables = Vec::with_capacity(agg_calls.len());
-        for (agg_call, ks) in agg_calls.iter().zip_eq(&keyspace) {
-            state_tables.push(generate_state_table(
```

Comment:
Can we remove `generate_state_table` now?

Reply:
It's now only used in unit test. I should add a test only tag

Reply:
Emmm I do not find one. Moved them all to test_utils.rs